lib/network.robot

```diff
@@ -68,7 +68,7 @@ Scan For Bluetooth In Linux
     ${out}=    Execute Linux Command    bluetoothctl power on
     Should Contain    ${out}    Changing power on succeeded
     Set DUT Response Timeout    60 seconds
-    Write Into Terminal    bluetoothctl scan on
+    Write Into Terminal    bluetoothctl -t 5 scan on
     Sleep    60s
     Write Bare Into Terminal    ${CTRL_C}
     ${out}=    Read From Terminal Until Prompt
```

Comment:
ok we still need `Read From Termin...` because we get the old ${out}
```
WLE003.001 Bluetooth scanning (Ubuntu) :: Check whether the Blueto... | FAIL |
'Changing power on succeeded' does not contain 'Discovery started'
```

Reply:
> because we get the old `${out}`

That's weird, not sure how it could get the same output second time.

> bare `bluetoothctl --timeout 60 scan on` has unfinished (i think) output

Maybe DUT response timeout should be less then `bluetoothctl`'s timeout?

Also if we're only interested in `Discovery started` in the output, 60 seconds seems like a lot.

Anyway, if it works reliably, this version should do.